docs/DeveloperGuide.md

```diff
@@ -126,7 +126,22 @@ The `Model` component,
 * stores a `UserPref` object that represents the user’s preferences. This is exposed to the outside as a `ReadOnlyUserPref` objects.
 * does not depend on any of the other three components (as the `Model` represents data entities of the domain, they should make sense on their own without depending on other components)
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Person` references. This allows `AddressBook` to only require one `Tag` object per unique tag, instead of each `Person` needing their own `Tag` objects.<br>
+
+
+<img src="images/ModelEventClassDiagram.png"  width="500"/>
+
+### Event Component
+
+An extension to the current Model is the `Event` Component,
+* Stores all information related to Event Objects, all of which are contained inside a `UniqueEventList` object.
+
+* Stores the current `Event` objects. Upon filtering via other commands (e.g: `calendar`), it is exposed to outsiders as an unmodifiable `ObservableList<Event>` that can be observed. 
```

Comment:
Nice noting that it is an unmodifiable list and hence defensively designed.

This is useful for reference in creating similar classes and for safe extensibility of the `Event` class!